web/src/components/navbar/desktop/NavBarDesktopQuickLinks.test.tsx

```diff
@@ -0,0 +1,84 @@
+import { NavBarDesktopQuickLinks } from "@/components/navbar/desktop/NavBarDesktopQuickLinks";
+import { getMergedConfig } from "@/contexts/configContext";
+import { fireEvent, render, screen } from "@testing-library/react";
+
+describe("<NavBarDesktopQuickLinks />", () => {
+  const Config = getMergedConfig();
+
+  window.open = jest.fn();
```

Comment:
dust - we've turned `open` into a jest function. Any reason we are not resetting mocks after each test. Any reason why we are not reverting the `open` fn back to what it previously was?

Reply:
I forgot is the reason